### ISSUE TITLE
How to run two formatters in sequence

### ISSUE THREAD

**Issue by lucatrv** [ryantam626/jupyterlab_code_formatter#97]
I would like to be able to run two formatters in sequence (e.g. `isort` and `black` in order). This applies to the following three use cases.

The first use case is by clicking on the toolbar button. For this I guess I should be able to define two formatters in the `default_formatter` field, which is not possible at present. The following setting gives a type error:

```js
{
    "preferences": {
        "default_formatter": {
            "python": [
                "isort",
                “black
            ]
        }
    }
}
```

The second use case is by using a keyboard shortcut, which I guess is not possible for the same reason.

The third use case is by always running the two formatters before saving the notebook. For this case, I wonder if I should open a separate issue on the `JupyterLab` repository itself.

**Comment by ryantam626** [comment]
I am aware of this :smile: since this is also my usual workflow. Just haven't sat down and implemented this, this should not be that complicated a change.

**Comment by ryantam626** [comment]
Hey apologies for getting back so late :sweat: 
This has now been implemented and available in plugin version 1.2.1 (this requires jupyterlab >=2.0.0), I have documented this feature in [here](https://jupyterlab-code-formatter.readthedocs.io/en/latest/how-to-use.html#chaining-default-formatters)

**Comment by lucatrv** [comment]
Thanks!!!
